examples/README.md

```diff
@@ -0,0 +1,21 @@
+# Eample usage of CDOGS
+
+If you want to use the KeyCloak Realm `jbd6rnxw` hosted by the Common Services Team (as used by this token endpoint https://dev.oidc.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token), you can request client setup here: https://getok.nrs.gov.bc.ca/app/about
```

Comment:
In markdown, you'll generally want to encapsulate raw http links with angle brackets (`<link>`) if you aren't using the `[text](link)` link format.